smart-contracts/contracts/mixins/MixinLockCore.sol

```diff
@@ -164,8 +164,8 @@ contract MixinLockCore is
   function updateBeneficiary(
     address _beneficiary
   ) external
+    onlyLockManager
```

Comment:
I thought we agreed on lockManager || beneficiary?

Reply:
I just caught that and renoved the request for review, but I was too late.